### ISSUE TITLE
Improve navbar accessibility by adding separate link for index (dashboard) page

### ISSUE THREAD

**Issue by KasperiP** [Testausserveri/testaustime-frontend#11]
I feel like current way to access dashboard / index page from other pages is not very intuitive. Would be more clear if there was a "dashboard"-link along friends & my profile links instead of clicking Testaustime "logo" (although its also just an anchor element).

**Comment by Eldemarkki** [comment]
Yes, this is something that should be addressed. What should the index page look like for logged in users, if /dashboard shows that dashboard page?

**Comment by KasperiP** [comment]
I mean the current index page is good. There is no need for moving it to /dashboard. What I was trying to say it that navbar should have a separate link for going back to index page from other pages instead of clicking "Testaustime"-logo. Anchor text for that could be "dashboard" for example. 

![image](https://user-images.githubusercontent.com/26027985/160252348-8e9c7ccf-d535-4e73-90c5-ace9f06f2273.png)
